### PR TITLE
Update contend.md

### PR DESCRIPTION
**Pull request by Fr4ng**
Added a sentence in Member Types to specify muliple admins allowed

## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/arduino-cloud/01.getting-started/09.arduino-cloud-for-business/contend.md [hunk 1]
```diff
@@ -120,6 +120,8 @@ There you will find the complete list of all members who have access to your Sha
 
 There are multiple role types in Arduino Cloud for Business to allow the admin to have full control of the rights/permissions each user has in the Shared Space. In this way, the admin can set up Role-Based-Access-Control (RBAC) by assigning profiles and sharing with any number of users.
 
+By default the member who has activated the plan and created the workspace is set as Admin. The Role-Based-Access-Control can support multiple Admins with same permissions.
```
Comment (martab1994) [inline content/arduino-cloud/01.getting-started/09.arduino-cloud-for-business/contend.md:123]: I agree with this sentence, but, at this point, I'd suggest to remove the sentence after the table saying "Thus, the only role with full permissions is the *Admin*, which corresponds to the owner of the Arduino Cloud for Business plan." If you agree, @Fr4ng could you do that in the same branch please?

**Review by martab1994 — CHANGES_REQUESTED** [review]

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Changes requested:
* Contemplate the possibility of removing the "thus" phrase

**Review by martab1994 — APPROVED** [review]
Perfect, we can merge thanks!

**Review by jcarolinares — APPROVED** [review]
Approved

waiting for linter checks and gatsnt preview

I will merge

**Comment by jcarolinares** [comment]
image
Looking good

**Pull request merged**